Jenkinsfile

```diff
@@ -1,7 +1,18 @@
-node("maven") {
-  checkout scm
-  stage("Build") {
-    sh "mvn fabric8:deploy -Popenshift -DskipTests"
-  }
-  stage("Deploy")
-}
+node('maven') {
+
+   stage ('Checkout') {
+        checkout scm
+    }
+   stage ('Standalone Integration Tests') {
+        sh "mvn clean verify"
+   }
+
+   stage ('OpenShift Integration Tests') {
+        sh "mvn clean verify -Popenshift-it,openshift,!standalone-it"
```

Comment:
OSO can not run even Jenkins master and slave. If we want to use `OpenShiftAssistant` for testing maybe we should enhance it with a support of another workspace for testing? (and store configuration of used workspaces somewhere ? e.g. config map)